2025-10-16 23:12:37,094	INFO monitor.py:698 -- Starting monitor using ray installation: D:\anaconda3\envs\metavobox_env\Lib\site-packages\ray\__init__.py
2025-10-16 23:12:37,094	INFO monitor.py:699 -- Ray version: 2.44.1
2025-10-16 23:12:37,094	INFO monitor.py:700 -- Ray commit: daca7b2b1a950dc7f731e34e74c76ae383794ffe
2025-10-16 23:12:37,094	INFO monitor.py:701 -- Monitor started with command: ['D:\\anaconda3\\envs\\metavobox_env\\Lib\\site-packages\\ray\\autoscaler\\_private\\monitor.py', '--logs-dir=D:\\Code\\dynamic_class/ray_tmp/session_2025-10-16_23-12-35_922386_54560\\logs', '--logging-rotate-bytes=536870912', '--logging-rotate-backup-count=5', '--gcs-address=127.0.0.1:49441', '--monitor-ip=127.0.0.1']
2025-10-16 23:12:37,098	INFO monitor.py:160 -- session_name: session_2025-10-16_23-12-35_922386_54560
2025-10-16 23:12:37,099	WARNING monitor.py:213 -- `prometheus_client` not found, so metrics will not be exported.
2025-10-16 23:12:37,100	INFO monitor.py:217 -- Monitor: Started
2025-10-16 23:12:37,106	INFO autoscaler.py:279 -- disable_node_updaters:False
2025-10-16 23:12:37,106	INFO autoscaler.py:287 -- disable_launch_config_check:True
2025-10-16 23:12:37,106	INFO autoscaler.py:299 -- foreground_node_launch:False
2025-10-16 23:12:37,106	INFO autoscaler.py:309 -- worker_liveness_check:True
2025-10-16 23:12:37,107	INFO autoscaler.py:359 -- StandardAutoscaler: {'cluster_name': 'default', 'max_workers': 0, 'upscaling_speed': 1.0, 'docker': {}, 'idle_timeout_minutes': 0, 'provider': {'type': 'readonly', 'use_node_id_as_ip': True, 'disable_launch_config_check': True}, 'auth': {}, 'available_node_types': {'ray.head.default': {'resources': {}, 'node_config': {}, 'max_workers': 0}}, 'head_node_type': 'ray.head.default', 'file_mounts': {}, 'cluster_synced_files': [], 'file_mounts_sync_continuously': False, 'rsync_exclude': [], 'rsync_filter': [], 'initialization_commands': [], 'setup_commands': [], 'head_setup_commands': [], 'worker_setup_commands': [], 'head_start_ray_commands': [], 'worker_start_ray_commands': []}
2025-10-16 23:12:37,108	INFO monitor.py:403 -- Autoscaler has not yet received load metrics. Waiting.
2025-10-16 23:12:42,111	INFO autoscaler.py:146 -- The autoscaler took 0.0 seconds to fetch the list of non-terminated nodes.
2025-10-16 23:12:42,111	INFO autoscaler.py:418 -- 
======== Autoscaler status: 2025-10-16 23:12:42.111332 ========
Node status
---------------------------------------------------------------
Active:
 1 node_8e2cf67ec3b685d76b7307ca6796d76d9bfca1988ebebd2c10e3aeeb
Pending:
 (no pending nodes)
Recent failures:
 (no failures)

Resources
---------------------------------------------------------------
Usage:
 18.0/18.0 CPU
 0B/2.38GiB memory
 302.24KiB/1.02GiB object_store_memory

Demands:
 (no resource demands)
2025-10-16 23:12:42,112	INFO autoscaler.py:461 -- The autoscaler took 0.001 seconds to complete the update iteration.
2025-10-16 23:12:47,115	INFO autoscaler.py:146 -- The autoscaler took 0.0 seconds to fetch the list of non-terminated nodes.
2025-10-16 23:12:47,115	INFO autoscaler.py:418 -- 
======== Autoscaler status: 2025-10-16 23:12:47.115493 ========
Node status
---------------------------------------------------------------
Active:
 1 node_8e2cf67ec3b685d76b7307ca6796d76d9bfca1988ebebd2c10e3aeeb
Pending:
 (no pending nodes)
Recent failures:
 (no failures)

Resources
---------------------------------------------------------------
Usage:
 18.0/18.0 CPU
 0B/2.38GiB memory
 302.24KiB/1.02GiB object_store_memory

Demands:
 (no resource demands)
2025-10-16 23:12:47,116	INFO autoscaler.py:461 -- The autoscaler took 0.001 seconds to complete the update iteration.
2025-10-16 23:12:52,118	INFO autoscaler.py:146 -- The autoscaler took 0.0 seconds to fetch the list of non-terminated nodes.
2025-10-16 23:12:52,118	INFO autoscaler.py:418 -- 
======== Autoscaler status: 2025-10-16 23:12:52.118582 ========
Node status
---------------------------------------------------------------
Active:
 1 node_8e2cf67ec3b685d76b7307ca6796d76d9bfca1988ebebd2c10e3aeeb
Pending:
 (no pending nodes)
Recent failures:
 (no failures)

Resources
---------------------------------------------------------------
Usage:
 18.0/18.0 CPU
 0B/2.38GiB memory
 302.24KiB/1.02GiB object_store_memory

Demands:
 (no resource demands)
2025-10-16 23:12:52,119	INFO autoscaler.py:461 -- The autoscaler took 0.001 seconds to complete the update iteration.
